### ISSUE TITLE
EKS access is doesn't working

### ISSUE THREAD

**Issue by vasili439** [harbur/kubernetic#100]
I've seen in changelog that since 2.3.0 kubernetic can access EKS clusters. But in my case no luck.

kubeconfig:

> apiVersion: v1
> clusters:
> - cluster:
>     certificate-authority-data: <cert>
>     server: https://MYID.sk1.eu-west-1.eks.amazonaws.com
>   name: dev-cluster.eu-west-1.eksctl.io
> contexts:
> - context:
>     cluster: dev-cluster.eu-west-1.eksctl.io
>     user: dev-cluster.eu-west-1.eksctl.io
>   name: dev-cluster.eu-west-1.eksctl.io
> current-context: dev-cluster.eu-west-1.eksctl.io
> kind: Config
> preferences: {}
> users:
> - name: dev-cluster.eu-west-1.eksctl.io
>   user:
>     exec:
>       apiVersion: client.authentication.k8s.io/v1alpha1
>       args:
>       - token
>       - -i
>       - dev-cluster
>       command: aws-iam-authenticator
>       env: null

if I try to browse EKS cluster with kubectl everything working fine:

> C:\Users\Vasily Yanov>kubectl get nodes
> NAME                                          STATUS    ROLES     AGE       VERSION
> ip-192-168-1-124.eu-west-1.compute.internal   Ready     <none>    44d       v1.12.7
> ip-192-168-66-73.eu-west-1.compute.internal   Ready     <none>    36d       v1.12.7`

Is it my mistake or is it a bug?

**Comment by vasili439** [comment]
Just fix it. In order to make it work you need to set AWS_PROFILE env var like:
`
.......
      command: aws-iam-authenticator
      env:
      - name: AWS_PROFILE
        value: user1
`

**Comment by spiddy** [comment]
weird, I have a different way to fix this:

Since Kubernetic is forking a process under the the Application, it doesn't know the PATH of your shell, so can't find the `aws-iam-authenticator` binary, instead provide the absolute path in the `command` field:

e.g.

``yaml
command: /usr/local/bin/aws-iam-authenticator
```

**Comment by spiddy** [comment]
Ok, so these are different problems, I understand in your case you're using a custom `AWS_PROFILE` which you probably inject in your shell and is not available at Kubernetic.

I also assume your aws-iam-authenticator is installed in a PATH that is part of the default PATHs that Kubernetic is looking at (I think it's /bin, /usr/bin/).

What is your OS? because in Mac [I got it installed] with brew (`brew install aws-iam-authenticator
`) and it landed on the `/usr/local/bin`

[I got it installed]: https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html
